pages/api/account.ts

```diff
@@ -19,7 +19,8 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
     try {
       await createAccountSchemaServer.validate(req.body, {context: {apiKey}})
     } catch (err) {
-      res.status(400).json({errors: err.errors})
+      const castedError = err as {error: string}
+      res.status(400).json({errors: castedError.error})
```

Comment:
What happens if `err` does not have an `error` string field?

The previous version returned `err.errors`, this one returns `err.error`. Did you intend to change this? Just want to make sure isn't a subtle bug here.

Reply:
cast as ValidationError from yup https://github.com/jquense/yup/blob/62786c42ca07a2b84b05ca8c473bc01f0c868a94/README.md#error-message-customization